idea-plugin/src/main/kotlin/arrow/meta/ide/testing/env/types/HeavyTestSyntax.kt

```diff
@@ -1,3 +1,6 @@
 package arrow.meta.ide.testing.env.types
 
+/**
+ * [HeavyTestSyntax] utilises common patterns in test environments for fully created ide instances
```

Comment:
quick note: with https://github.com/arrow-kt/arrow-meta/pull/252 it'll be (probably) be possible to drop heavy tests

Reply:
Yes, for sure I'd like to keep it, because #252 won't be merged before KotlinConf.